### ISSUE TITLE
Change normalization stats of a pretrained checkpoint

### ISSUE THREAD

**Issue by stevenoh2003** [Physical-Intelligence/openpi#546]
I would like to use the pretrained DRIOD checkpoints on my UR robot. I notice that in the documentation you mention that we can use the robot specific "asset" to normalize the action space of the train dataset. Could I directly do this on a pretrained DRIOD checkpoint? Or do I have to retrain a checkpoint and add the normalization?

**Comment by kvablack** [comment]
It is very, very unlikely that our models will work on your robot configuration without some fine-tuning. I would recommend fine-tuning on data from your own robot and using that same data to compute norm stats.

**Comment by stevenoh2003** [comment]
Thank you for your reply. I have been experimenting loading the existing asset (in my case for my UR5 robot) vs computing my own asset and loading it.
```

assets=AssetsConfig(
   assets_dir="s3://openpi-assets/checkpoints/pi0_base/assets",
   asset_id="ur5e"),
```

I was wondering how would I change this to a local asset directory? Could you clafiy as i could not find it in the documentation. Thanks!
